cmd/celestia-appd/cmd/download-genesis.go

```diff
@@ -0,0 +1,118 @@
+package cmd
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/spf13/cobra"
+)
+
+var chainIDToSha256 = map[string]string{
+	"celestia":   "9727aac9bbfb021ce7fc695a92f901986421283a891b89e0af97bc9fad187793",
+	"mocha-4":    "0846b99099271b240b638a94e17a6301423b5e4047f6558df543d6e91db7e575",
+	"arabica-10": "fad0a187669f7a2c11bb07f9dc27140d66d2448b7193e186312713856f28e3e1",
+}
+
+func downloadGenesisCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "download-genesis chain-id",
+		Short: "Download genesis file from https://github.com/celestiaorg/networks",
+		Long: "Download genesis file from https://github.com/celestiaorg/networks.\n" +
+			"The first argument must be a known chain-id. Ex. celestia, mocha-4, or arabica-10.\n",
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			chainID := args[0]
+			if !isKnownChainID(chainID) {
+				return fmt.Errorf("unknown chain-id: %s. Must be: celestia, mocha-4, or arabica-10", chainID)
+			}
+			outputFile := server.GetServerContextFromCmd(cmd).Config.GenesisFile()
+			fmt.Printf("Downloading genesis file for %s to %s\n", chainID, outputFile)
+
+			url := fmt.Sprintf("https://raw.githubusercontent.com/celestiaorg/networks/master/%s/genesis.json", chainID)
+			if err := downloadFile(outputFile, url); err != nil {
+				return fmt.Errorf("error downloading genesis file: %s", err)
```

Comment:
[Optional-nit]I would differentiate the error messages returned by this function. Users could receive distinct prompts: one for issues related to file creation and another for downloading problems. 

Alternatively, the following message could be changed to better reflect both conditions simultaneously, e.g.,
```suggestion
				return fmt.Errorf("error downloading / persisting the genesis file: %s", err)
```